src/OrchardCore.Cms.Web/Startup.cs

```diff
@@ -1,14 +1,19 @@
 using Microsoft.AspNetCore.Builder;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using OrchardCore.MiniProfiler;
 
 namespace OrchardCore.Cms.Web
 {
     public class Startup
     {
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddOrchardCms();
+            services.AddOrchardCms(builder => builder.ConfigureServices(services =>
+                services.PostConfigure<MiniProfilerOptions>(settings =>
```

Comment:
No extension method ;)

I think you left some testing code in here?

Maybe you could add this bit to the docs, the Configuration docs don't actually talk about configuring services in the builder, just how `appsettings.json` works (and we don't have any docs that do, so it's super helpful to at least add it somewhere, as a start)

Reply:
Yeah, I managed to commit the testing code this time too, thanks for pointing it out. I was going to come back and add an extension, occurred to me during shower :). Done both.

I've also added more docs.